platform/osgi/platform-osgi-conditions/pom.xml

```diff
@@ -48,6 +48,14 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <configuration>
+                    <!--Plugin wants to remove imports that should be there-->
```

Comment:
It fails because checkstyle is broken for imports for comments and always has been. I'm not going to fix it for this PR, but we should probably just disable everything but the header check since it is already being done by the Google formatter.